ipa-core/src/net/server/mod.rs

```diff
@@ -103,6 +108,21 @@ impl MpcHelperServer<Helper> {
     }
 }
 
+impl MpcHelperServer<Shard> {
+    #[must_use]
+    pub fn new_shards(
+        _transport: &ShardHttpTransport,
+        config: ServerConfig,
+        network_config: NetworkConfig<Shard>,
+    ) -> Self {
+        MpcHelperServer {
```

Comment:
This seems incomplete, is it temporary similar to the TODO above? 

```
// TODO: Following is just temporary until Shard Transport is actually used.
```

Reply:
Correct. This will be filled once I push the changes for the Shard routes.